components/studentPortal/layout/Sidebar.tsx

```diff
@@ -126,6 +175,7 @@ export const Sidebar: React.FC = () => {
               <path d="M5 3a2 2 0 00-2 2v2a2 2 0 002 2h2a2 2 0 002-2V5a2 2 0 00-2-2H5zM5 11a2 2 0 00-2 2v2a2 2 0 002 2h2a2 2 0 002-2v-2a2 2 0 00-2-2H5zM11 5a2 2 0 012-2h2a2 2 0 012 2v2a2 2 0 01-2 2h-2a2 2 0 01-2-2V5zM11 13a2 2 0 012-2h2a2 2 0 012 2v2a2 2 0 01-2 2h-2a2 2 0 01-2-2v-2z" />
             </svg>
           }
+          userRole={userRole}
```

Comment:
sidebarItems don't need the userRole. UserRole is kind of a sensitive property that we don't want to make accessible everywhere unless absolutely needed. 

Rather than passing in userRole, I would just pass in isDisabled={userRole === "freemium"}